app/src/main/res/values/themes.xml

```diff
@@ -74,6 +74,20 @@
         <item name="android:colorAccent">@color/blue</item>
     </style>
 
+    <style name="AlertDialogTheme" parent="Theme.AppCompat.DayNight.Dialog.Alert">
+        <item name="buttonBarNegativeButtonStyle">@style/NegativeButtonStyle</item>
+        <item name="buttonBarPositiveButtonStyle">@style/PositiveButtonStyle</item>
+        <item name="android:background">@color/dialog_background</item>
+    </style>
+
+    <style name="NegativeButtonStyle" parent="Widget.AppCompat.Button.ButtonBar.AlertDialog">
```

Comment:
please move view styles to styles.xml (NegativeButtonStyle, PositiveButtonStyle)